.ci/azure/windows.yml

```diff
@@ -109,6 +111,20 @@ jobs:
     workingDirectory: $(WORK_DIR)
     displayName: 'Install dependencies'
 
+  - script: |
+      $(PYTHON_EXE) -m pip install build
+      call $(SETUPVARS) -pyver 3.8 && $(PYTHON_EXE) -m  build --wheel --outdir $(BUILD_WHEEL_DIR_CONTRIB) $(REPO_DIR)\modules\custom_operations
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Build tokenizers wheel'
+
+  - script: |
+      $(PYTHON_EXE) -m pip install $(BUILD_WHEEL_DIR_CONTRIB)/*.whl
+      workingDirectory: $(WORK_DIR)
+    displayName: 'Install tokenizers wheel'
+
+  - publish: $(BUILD_WHEEL_DIR_CONTRIB)
+    artifact: Wheels
```

Comment:
looks like we need to move it below, because we need to build OpenVINO first

Reply:
done